README.md

```diff
@@ -1,3 +1,29 @@
+````
+-----------------------------          _______  _______  _______  _______ ------------------------------
+-----------------------------|\     /|(  ____ \(  ___  )(  ____ \(  ____ \------------------------------
+-----------------------------| )   ( || (    \/| (   ) || (    \/| (    \/------------------------------
+-----------------------------| |   | || (__    | |   | || (_____ | (_____ ------------------------------
+-----------------------------( (   ) )|  __)   | |   | |(_____  )(_____  )------------------------------
+----------------------------- \ \_/ / | (      | |   | |      ) |      ) |------------------------------
+-----------------------------  \   /  | )      | (___) |/\____) |/\____) |------------------------------
+-----------------------------   \_/   |/       (_______)\_______)\_______)------------------------------
+                                   Vanderbilt Free & Open Source Society
```

Comment:
Should we just put the actual logo here?

Reply:
ascii is cool